charts/nautobot/values.yaml

```diff
@@ -236,6 +235,23 @@ nautobot:
             value: 2
             periodSeconds: 15
 
+  # Add a Persistent Volume Claim for /static files of Nautobot
+  persistence:
+    # --- Enable PVC and the relevant volumes
+    enabled: false
+    # --- [Kubernetes StorageClass Name](https://kubernetes.io/docs/concepts/storage/storage-classes/), for the creation of Persistent Volume Claim
+    storageClass: ""
+    # --- [Kubernetes Volumes AccessMode](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+    accessMode: "ReadWriteOnce"
+    # --- Persistent storage size request
+    size: "2Gi"
```

Comment:
Perhaps we can make this volume even smaller by default.

Reply:
Made it `1Gi`